src/main/java/com/example/moviemanager/service/tmdbmovieservice/configuration/ConfigProperties.java

```diff
@@ -0,0 +1,36 @@
+package com.example.moviemanager.service.tmdbmovieservice.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "tmdb")
+public class ConfigProperties {
```

Comment:
Even prefix says that it is tmdb properties, so name it accordingly. Because ConfigProperties is very abstract